src/components/app/ProfileRootMessage.js

```diff
@@ -26,12 +27,16 @@ export class ProfileRootMessage extends React.PureComponent<Props> {
     return (
       <div className="rootMessageContainer">
         <div className="rootMessage">
-          <h1 className="rootMessageTitle">Firefox Profiler</h1>
+          <Localized id="profileRootMessage--title">
+            <h1 className="rootMessageTitle">Firefox Profiler</h1>
+          </Localized>
           <div className="rootMessageText">{message}</div>
           {additionalMessage ? (
             <div className="rootMessageAdditional">
               {this.toParagraphs(additionalMessage)}
-              <a href="/">Back to home</a>
+              <Localized id="profileRootMessage--additional">
+                <a href="/">Back to home</a>
+              </Localized>
```

Comment:
I couldn't test this line. Not sure if it's the correct way to wrap <a /> tags. For another one I had to include elems attribute in the wrapper becuase that was nested inside <p /> tag.

Reply:
There is nothing in the href attribute to localize. I think it should be fine. Will check it manually too.
I think you can also manually test it by disconnecting from the internet and then trying to load a "public" profile.